apps/optimizely/src/EditorPage.spec.js

```diff
@@ -13,6 +13,9 @@ describe("EditorPage", () => {
   beforeEach(() => {
     sdk = mockProps.sdk;
     sdk.entry.fields.experimentId.onValueChanged = fn => () => {};
+
+    Date.now = jest.fn(() => 100);
```

Comment:
Is there a reason for this change? 🤔

Reply:
Tests failed due to tricky date and random number mock logic inside the Dropdown component:
https://github.com/contentful/forma-36/blob/99609176a0604fd0f6b62753cf45a246cdbb48b6/packages/forma-36-react-components/src/components/Dropdown/Dropdown.test.tsx#L11-L14

Ids are generated based on random number, so needed to stub it here too. Checked this in with @suevalov

Reply:
Good to know, thanks!